### ISSUE TITLE
test result

### ISSUE THREAD

**Issue by chenkaixin66** [xuuuluuu/Aspect-Sentiment-Classification#2]
I reproduced the code you provided, and the test set result did not reach the value written in the paper. Why? Did I do something wrong?
The acc result that I reproduced on the rest15 data set is 76.91 and your paper is 80.82.There are gaps in every data set

**Comment by xuuuluuu** [comment]
Are you able to reproduce the results with our saved model?

**Comment by chenkaixin66** [comment]
> Are you able to reproduce the results with our saved model?

I can reproduce the results with your saved model. But I am a Windows system and cannot open the tar file. I want to know the parameters of the best model you have saved. Can you provide me with them?

**Comment by chenkaixin66** [comment]
> > Are you able to reproduce the results with our saved model?
> 
> I can reproduce the results with your saved model. But I am a Windows system and cannot open the tar file. I want to know the parameters of the best model you have saved. Can you provide me with them?

What hardware equipments do you use？

**Comment by xuuuluuu** [comment]
Hi, note that we split a dev set from the training set, which allows us to conduct a grid search of the hyperparameters. The ranges of the parameters are listed in Section 3.1 of the paper. 

We used a single 2080Ti for the experiments.

**Comment by xuuuluuu** [comment]
For a quick test, you can check lines 131-144 in the  train_crf_glove.py for the hyperparameter of different datasets based on my hardware settings.

**Comment by chenkaixin66** [comment]
> For a quick test, you can check lines 131-144 in the train_crf_glove.py for the hyperparameter of different datasets based on my hardware settings.

Thank you . In the AspectSent class defined in model_batch_crf_glove.py in your source code, self.conv and self.convs are written in the AspectSent class, but you also commented that # cnn-not used for current model I want to know if this is running during training ? What is this function?

**Comment by xuuuluuu** [comment]
Hi, you can refer to the code that these two functions are not called during training. The functions were used for my initial trial experiments. You may ignore them.
